eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -0,0 +1,141 @@
+# REST API v3: content management
+
+This specification defines the next iteration of the REST API in eZ Platform known as V3. The API aims to be backwards compatible but will also introduce new capabilities.
+
+As a developer the overall vision of V3 is that the API should create simpler endpoints that can get me started quicker with the API. The endpoints will more default to values like language, draft, versions and misc options in order to make the threshold to use the API lower.
+
+The overall vision is: simple by default and still making the advanced capabilities efficient.
+
+The goal of the API defined here is to manage object fetch, create update and deletion. For read only and content heavy uses the GraphQL endpoints should be used.
```

Comment:
@DavidLiedle this does not need to be a requirement. Theoretically by evolving our current API by embracing some aspects of JSON API like [includes](http://jsonapi.org/format/#fetching-includes) and [sparse fieldsets](http://jsonapi.org/format/#fetching-sparse-fieldsets) we have the features needed by UI to be able to get the data it need without having to involve GraphQL.

Aka, prototyping would be good here to be able to compare before we make such a technical decision.